pkg/controllers/dynakube/extension/utils/naming.go

```diff
@@ -0,0 +1,18 @@
+package utils
```

Comment:
IMO better to not use utils, and name package based on content: something like service or service_name.

Reply:
potentially you can just add few extra methods to DK, since it's just few consntannts and dk instance, but make sure you don't have any import problems.

Reply:
funny as I thought about it before doing it. We have consts package, but there it doesn't fit either, as a func in consts just feels wrong.

if we would name it definitions or defs or something we could place it there...

Reply:
you didn't get the point, it's okey have lot's of small packages in go, just rename utils to something meaningful.
For example: `service`, then we can easily use `service.BuildName` and `service.BuildFQDName`, because package name is always part of API.

Reply:
I got your point, but did not want to touch so many files for this purpose.
But your suggestion to rename `util` to `service` is a good one as the functions are all related to service creation.

fixed in 187d5ef6e57ad5f248c05eec21f36fc72c0c24c9